services/proxy/pkg/config/config.go

```diff
@@ -108,6 +108,8 @@ type OIDC struct {
 	UserinfoCache           *Cache `yaml:"user_info_cache"`
 	JWKS                    JWKS   `yaml:"jwks"`
 	RewriteWellKnown        bool   `yaml:"rewrite_well_known" env:"PROXY_OIDC_REWRITE_WELLKNOWN" desc:"Enables rewriting the /.well-known/openid-configuration to the configured OIDC issuer. Needed by the Desktop Client, Android Client and iOS Client to discover the OIDC provider."`
+	ClientID                string `yaml:"client_id" env:"OCIS_OIDC_CLIENT_ID;PROXY_OIDC_CLIENT_ID" desc:"OIDC client ID, which ownCloud Web uses. This client needs to be set up in your IDP."`
+	SkipClientIDCheck       bool   `yaml:"skip_client_id_check" env:"PROXY_OIDC_SKIP_CLIENT_ID_CHECK" desc:"If true will skip checking the configured client id is present in audience claims."`
```

Comment:
@mmattel could you do your magic with these strings?

Reply:
Questions:
* `PROXY_OIDC_CLIENT_ID` --> This client needs to be set up in your IDP.
   - What is necessary to be done in a foreign IDP?
   - As we have an embedded IDP, what do we need to do there?
   - Where does one get the (our) web client ID?
* `PROXY_OIDC_SKIP_CLIENT_ID_CHECK`  --> ... client id is present in audience claims.
Is `audience claim` a well known term?